verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -238,8 +245,8 @@ VName KytheFactsExtractor::ExtractDataTypeReference(
   const auto& anchors = data_type_reference.Value().Anchors();
   const Anchor& type = anchors[0];
 
-  const VName type_vname = *ABSL_DIE_IF_NULL(
-      scope_context_.SearchForDefinition(CreateModuleSignature(type.Value())));
+  const VName type_vname =
+      *ABSL_DIE_IF_NULL(scope_context_.SearchForDefinition(type.Value()));
```

Comment:
Will this crash on invalid/incomplete code?  Should this cause some fail-safe behavior instead?

Reply:
Still working on it.